source/devel_docker_ci.rst

```diff
@@ -0,0 +1,255 @@
+=============================
+Docker CI with GitHub Actions
+=============================
+
+.. _Docker Hub: https://hub.docker.com/
+.. _Docker: https://www.docker.com/
+.. _GitHub: https://github.com/
+.. _Continuous integration: https://en.wikipedia.org/wiki/Continuous_integration
+.. _GitHub Actions Documentation: https://help.github.com/en/actions
+.. _GitHub Actions: https://github.com/features/actions
+.. _Overview of Docker Compose: https://docs.docker.com/compose/
+.. _Automated repository tests: https://docs.docker.com/docker-hub/builds/automated-testing/
+.. _Semantic Versioning 2.0.0: https://semver.org/
+.. _Travis CI: https://travis-ci.org/
+.. _Coveralls: https://coveralls.io/
+
+All docker containers, and all code should be tested. `Continuous integration`_, CI, with `GitHub Actions`_ takes care of this for repositories that has a Docker_ container. Repositories that comes without a Docker container can be tested with, e.g., `Travis CI`_ and Coveralls_ for monitoring test coverage.
```

Comment:
we may need to change the last sentence here but I suggest we do that after your review on the mmd repos'es

Reply:
GitHub do have GitHub Actions that integrate right into Coveralls. I understand the CI bit, but need help from a developer to create a test suite. If we go for that solution I think we can keep the text.